fin/CMakeLists.txt

```diff
@@ -63,8 +63,8 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 endif()
 
 ############################################################
-# require C++17
-add_compile_options(-std=c++17)
+# require C++14
+add_compile_options(-std=c++14)
```

Comment:
MIOpen uses C++17. Why do you need C++14?

Reply:
This is in Fin's CmakeList and cpp 17 was crashing the build there.